core/src/main/java/hudson/model/DownloadService.java

```diff
@@ -401,7 +401,7 @@ public FormValidation updateNow() throws IOException {
             for (UpdateSite updatesite : Jenkins.getActiveInstance().getUpdateCenter().getSiteList()) {
                 String site = updatesite.getMetadataUrlForDownloadable(url);
                 if (site == null) {
-                    return FormValidation.warning("The update site " + site + " does not look like an update center");
+                    return FormValidation.warning("The update site " + updatesite + " does not look like an update center");
```

Comment:
Could you please show us an example of the resulting warning?

Reply:
Maybe updatesite.getId() could be used?

Reply:
Thanks for the suggestion. I used getId() instead.